styleguide/source/assets/scss/02-molecules/_people-listing.scss

```diff
@@ -50,7 +50,6 @@
   &__details {
     @include gutter-all($padding-all-half...);
     border-top: 1px solid $gray-50;
-    word-break: break-all;
```

Comment:
I wouldn't recommend remove this because this was added when names were really long or urls were really long. I would instead use break-word